modules/attributes/pages/positional-and-named-attributes.adoc

```diff
@@ -56,3 +58,22 @@ No special processing is performed, except for attribute reference expansion, if
 
 If the attribute value contains the same quote character being used to enclose the value, escape the quote character in the value by prefixing it with a backslash (e.g., `title="\"Dark Horse\" is a song by George Harrison"`).
 // end::subs[]
+
+== Attribute parsing
+
+The exact rules for determining the boundaries of attributes and whether an attribute is positional or named are somewhat complicated.
+
+A `name` consists of a word character (letter or numeral) followed by any number of word characters, `.` charactes, or `-` characters, followed by something else (including the end of the string)
+
+* Parsing an attribute proceeds from the beginning of the attribute list string or after a previously recognized delimiter (`,`).
+* Blank space is skipped.
+* If there is a name, followed by optional blank space, followed by `=`, then a named attribute is recognized, with the just-read name.
+Parsing the value proceeds after skipping blank space after the `=`.
```

Comment:
I think too much attention is given here to the space around the equals sign. Instead, it should read as:

> If a valid attribute name is found and is followed by an equals sign (`=`), then the parser recognizes this as a named attribute. The text after the equals sign (`=`) and up to the next comma or end of list is taken as the attribute value. Spaces around the equals sign (`=`) are ignored.